scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -0,0 +1,31 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+    using MiniJSON;
+
+    public class ShopifyError {
+        public enum ErrorType {
+            HTTP, GraphQL
+        }
+
+        public readonly ErrorType error;
+        public readonly string description;
+
+        public ShopifyError(ErrorType error, string description) {
+            this.error = error;
+            this.description = description;
+        }
+
+        public static explicit operator ShopifyError(TopLevelResponse response) {
```

Comment:
Can this `static` method be placed at the top of the file. This is typically what's been done in other files.

```
Public Static Variables
Private Static Variables
Public Static methods
Private Static Methods
Public Property definitions
Private Property definitions
Constructor
Public Methods
Private Methods
```